recipe-client-addon/lib/Storage.jsm

```diff
@@ -29,6 +30,34 @@ function loadStorage() {
 }
 
 this.Storage = {
+  DURABILITY_NAMESPACE: '_storageDurability',
+  DURABILITY_KEY: 'durable',
+  async seedDurability(sandbox) {
+    const globalDurabilityStore = Storage.makeStorage(this.DURABILITY_NAMESPACE, sandbox);
```

Comment:
What if an action tries to calls `makeStorage` with the same prefix? Is that something we want to guard against? Or should we just accept that a prefix named "durability" is pretty clear about what it is doing?